### DISCUSSION TITLE
Payment

### DISCUSSION THREAD

**Discussion by Mivc32** [d4rken-org/sdmaid-se#699] [Q&A]
Hi,

I already had SDMAID PRO which I paid for. Do I also  have to pay for the new SDMAID SE Or an update price?

Thanks

**Comment by d4rken** [comment]
It's an extra app with its own purchase, see:
https://github.com/d4rken-org/sdmaid-se/wiki/FAQ#transfer-pro-status-from-sd-maid-1
